src/openrct2/actions/RideSetSettingAction.cpp

```diff
@@ -215,6 +215,9 @@ GameActions::Result RideSetSettingAction::Execute() const
             {
                 ride->lifecycleFlags |= RIDE_LIFECYCLE_MUSIC;
             }
+            // Set invalidation flag to trigger window redraw
+            // https://github.com/OpenRCT2/OpenRCT2/issues/24589
```

Comment:
I think the code is self explanatory and we typically don't reference issues unless its on-going. If one must find the reason to why the line exists its better to use tools like git blame to do that.